### ISSUE TITLE
Root fragment breaks build

### ISSUE THREAD

**Issue by fs-c** [kartiknair/dhow#7]
A MWE is
```js
import Dhow from 'dhow';

const Page = () => (
    <><p>hello</p></>
);

export default Page;
```
, it breaks with the following error:
```
TypeError: Cannot read property 'length' of undefined
    at serializeElement (/home/lw/Projects/dhow/node_modules/min-document/serialize.js:32:29)
```

This is a pretty common pattern, in part since it allows doing stuff like
```js
const Page = () => {
    <Head></Head>

    <header></header>

    <main></main>

    <footer></footer>
};
```
without having to use unnecessary/semantically wrong elements.

**Comment by fs-c** [comment]
PR is at #8

**Comment by kartiknair** [comment]
Thanks for opening the issue & the PR! I've published the patch to NPM, so the fixed version should be live on `dhow@3.0.4`.
